maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -336,4 +339,15 @@ object MaestroSessionManager {
 
     }
 
+    private class CustomSignalHandler() : SignalHandler {
+        override fun handle(signal: Signal) {
+            when (signal) {
+                suspendSignal -> exitProcess(0)
```

Comment:
Will this trigger a shutdown hook on line 86?

Reply:
Yes. But since ctrl+z suspend the main process, so it will run in the background